adam-cli/src/main/scala/org/bdgenomics/adam/cli/Flatten.scala

```diff
@@ -1,82 +0,0 @@
-/**
```

Comment:
We don't use it anymore, we had a ticket (#1244) that was leaning towards removing it, and it started causing unit test failures (due to an issue I was hinting at in said ticket), so I went ahead and just got rid of it here.